docs/achtergronddocumentatie/artefacten/1304.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: page-with-side-nav
+title: B1304 - Resource Partij met contactpersoon, organisatie en persoon. 
+date: 02-10-2023
+---
+
+# B1304 - Resource Partij met contactpersoon, organisatie en persoon. 
+
+## Besluit
+
+De objecttypen Part, Persoon, Contactpersoon en Organisatie wordt samengevoegd in de resource Partij. 
```

Comment:
Wat is het objecttype 'Part'? En wordt hier bewust onderscheid gemaakt tussen objecttypen en resources?

Reply:
Objecttype Part is een typefout. Aangepast.

Er is bewust onderscheid tussen objecttypen (in het informatiemodel) en Resources (in het API-schema) gemaakt. Daar kunnen verschillen tussen zitten en die licht ik hier toe. 

Ik worstel nog wel een beetje met de terminologie voor het UGM. Daarin heten het nu entiteittypen. Dat komt volgens mij overeen met de resources.

Reply:
Ah, nu snap ik het: Part moest Partij zijn (had ik écht niet door) en alle vier worden samengevoegd in UGM-resource of entiteittype Partij. Helder. Misschien ter verduidelijking expliciet verwijzen naar SIM en UGM?

Reply:
Done